ssl/statem/statem_lib.c

```diff
@@ -277,11 +277,8 @@ MSG_PROCESS_RETURN tls_process_cert_verify(SSL *s, PACKET *pkt)
 {
     EVP_PKEY *pkey = NULL;
     const unsigned char *data;
-#ifndef OPENSSL_NO_GOST
-    unsigned char *gost_data = NULL;
-#endif
```

Comment:
Exactly. I also didn't notice this issue. So moving the gost_data declaration to the original position.